teradatalabs/mapr52-hive/files/bootstrap.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# CONFIGURE MapR
+/opt/mapr/server/configure.sh  -N mycluster -Z localhost -C localhost -HS localhost -no-autostart
+
+# SETUP FLAT FILE /home/mapr/storagefile
+dd if=/dev/zero of=/home/mapr/storagefile bs=1G count=10
```

Comment:
I'm not sure if this should be done here or in a setup.sh script. Probably depends on image size and the container startup time